packages/arb-ts/src/lib/message/L1ToL2MessageCreator.ts

```diff
@@ -78,7 +78,7 @@ export class L1ToL2MessageCreator {
     )
     const l2ChainID = (await l2Provider.getNetwork()).chainId.toString()
     const rec = await this.createRetryableTicketFromGasParams(
-      gasParams,
+      { ...gasParams, l2CallValue },
```

Comment:
A bit confused as to what happened here — was it just erroring at the static checker level before this?

Reply:
No, I'm just adding in the l2CallValue. It was part of the return from `estimateGasValuesL1ToL2Creation` but I removed that as it wasnt generated in there I think